build/Build.InstallationScripts.cs

```diff
@@ -0,0 +1,22 @@
+using Nuke.Common;
+using Nuke.Common.IO;
+
+partial class Build : NukeBuild
+{
+    readonly AbsolutePath InstallationScriptsFolder = RootDirectory / "InstallationScripts";
+
+    Target BuildInstallationScripts => _ => _
+        .After(Clean)
+        .Executes(() =>
+        {
+            var scriptTemplates = RootDirectory / "script-templates";
+            var templateFiles = scriptTemplates.GetFiles();
+            foreach (var templateFile in templateFiles)
+            {
+                var scriptFile = InstallationScriptsFolder / templateFile.Name.Replace(".template", "");
+                FileSystemTasks.CopyFile(templateFile, scriptFile);
+                scriptFile.UpdateText(x =>
```

Comment:
You can leave a comment that we could use https://github.com/scriban/scriban if more replaces would be needed.